swift/extractor/infra/TargetDomains.h

```diff
@@ -12,6 +12,7 @@ enum class TrapType {
   module,
   invocation,
   linkage,
+  lazy_declarations,
```

Comment:
cosmetic nit for consistency with other consts:
```suggestion
  lazy_declaration,
```